sbol2/component.py

```diff
@@ -17,8 +17,8 @@ def __init__(self, rdf_type, uri, definition, access, version):
         self.access = URIProperty(self, SBOL_ACCESS, '0', '1', [], access)
         self.mapsTos = OwnedObject(self, SBOL_MAPS_TOS, MapsTo,
                                    '0', '*', [])
-        self.measurements = OwnedObject(self, SBOL_MEASUREMENTS, Measurement,
-                                        '0', '*', [])
+        self.measures = OwnedObject(self, SBOL_MEASUREMENTS, Measure,
```

Comment:
Also maybe change the constant name from SBOL_MEASUREMENTS -> SBOL_MEASURES?

Reply:
No need, we are back to measurements. "measures" is gone.